crates/air_r_formatter/src/r/auxiliary/call.rs

```diff
@@ -32,12 +31,15 @@ impl FormatNodeRule<RCall> for FormatRCall {
     }
 
     fn is_suppressed(&self, node: &RCall, f: &RFormatter) -> bool {
-        has_skip_comment(node.syntax(), f) || in_skip_setting(node, f).unwrap_or(false)
+        let comments = f.comments();
+        comments.mark_suppression_checked(node.syntax());
+        comments.has_skip_directive(node.syntax()) || in_skip_setting(node, f).unwrap_or(false)
     }
```

Comment:
Again here's that general pattern of an `is_suppressed()` override doing both `mark_suppression_checked()` and `has_skip_directive()`, with an additional check for the `skip` configuration setting since this is a call